app/models/user.rb

```diff
@@ -1,2 +1,4 @@
 class User < ApplicationRecord
+  has_many :comments
+  has_many :posts, foreign_key: 'author_id'
```

Comment:
- [ ] following the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) the user has many likes as well kindly fix that :pray: 

example : 

```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments
has_many :likes
```